content/en/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -305,7 +305,7 @@ Noteworthy points about the nginx-secure-app manifest:
   serves HTTP traffic on port 80 and HTTPS traffic on 443, and nginx Service
   exposes both ports.
 - Each container has access to the keys through a volume mounted at `/etc/nginx/ssl`.
-  This is setup *before* the nginx server is started.
+  This is set up *before* the nginx server is started.
```

Comment:
```suggestion
  This is setup *before* the nginx server is started.
```

Reply:
This is using the verb form, so the correct sentence is:
"This is set up *before* the nginx server is started."

If this were being used as a noun, the sentence would be:
"This setup is *before* the nginx server is started."

Which form would you prefer?

Reply:
If you are not sure, please keep unchanged when the original is fine.

Reply:
The original has incorrect grammar;
But I'm removing this fix from my PR because there is no agreement on how to correct it.